src/analysis/calculus/deriv.lean

```diff
@@ -1670,6 +1672,41 @@ by simp only [div_eq_mul_inv, deriv_mul_const_field]
 
 end division
 
+section star
+/-! ### Derivative of `x ↦ star x` -/
+
+variables [star_ring 𝕜] [has_trivial_star 𝕜] [star_add_monoid F] [has_continuous_star F]
+variable [star_module 𝕜 F]
+
+theorem has_deriv_at_filter.star (h : has_deriv_at_filter f f' x L) :
```

Comment:
Please make all these theorems `protected`, otherwise in Lean 4
```lean
theorem has_deriv_at.something : star x = y
```
will try to interpret `star` as `has_deriv_at.star`.

Reply:
Done